### PR TITLE
Bug 1159660 - to check if activeElement is itself before calling focus o...

### PR DESCRIPTION
**Pull request by huchengtw-moz**
...r blur a mozbrowser iframe

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=51a6a999cc5a4bb0654231dbdae38263b9dd9357

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cae352e49ebf887cdc9b4bba0229e0922d99d36c

File: apps/system/js/browser_mixin.js [hunk 1]
```diff
@@ -173,16 +173,22 @@
       }
     },
```
Comment (alivedise) [inline apps/system/js/browser_mixin.js]: Make a comment here
// For testing purpose; we cannot mock document.activeElement in unit test...

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0ab56c16152011b5d4ef8d3f75730a92dd2feb34

**Pull request merged**